functions/graphql/index.js

```diff
@@ -64,7 +64,7 @@ function configureServer() {
     uploads: false,
     context: async ({ req }) => {
       // PRODUCTION VERSION
-      const token = (req.headers.authorization || "").replace("Bearer ", "");
+      // const token = (req.headers.authorization || "").replace("Bearer ", "");
```

Comment:
Make sure this is uncommented so it doesn't get broken for me.
```suggestion
      const token = (req.headers.authorization || "").replace("Bearer ", "");
```